File: packages/java-sdk/src/main/java/dev/blindfold/sdk/models/TokenizeResponse.java
package dev.blindfold.sdk.models;

import com.google.gson.annotations.SerializedName;
import java.util.List;
import java.util.Map;

public class TokenizeResponse {
    private String text;
    private Map<String, String> mapping;

    @SerializedName("detected_entities")
    private List<DetectedEntity> detectedEntities;

    @SerializedName("entities_count")
    private int entitiesCount;

    public TokenizeResponse() {}

    public String getText() { return text; }
    public Map<String, String> getMapping() { return mapping; }
    public List<DetectedEntity> getDetectedEntities() { return detectedEntities; }
    public int getEntitiesCount() { return entitiesCount; }
}

File: packages/dotnet-sdk/src/Blindfold.Sdk/Models/TokenizeResponse.cs
using System.Text.Json.Serialization;

namespace Blindfold.Sdk.Models;

public class TokenizeResponse
{
    [JsonPropertyName("text")]
    public string Text { get; set; } = "";

    [JsonPropertyName("mapping")]
    public Dictionary<string, string> Mapping { get; set; } = new();

    [JsonPropertyName("detected_entities")]
    public List<DetectedEntity> DetectedEntities { get; set; } = new();

    [JsonPropertyName("entities_count")]
    public int EntitiesCount { get; set; }
}
